defaults/aws_config.yaml

```diff
@@ -10,22 +10,22 @@ instance_type_monitor: 't3.large'
 regions_data:
   us-east-1: # US East (N. Virginia)
     ami_id_loader: 'ami-00c1a16b5136fbb7c' # Loader dedicated AMI v19
-    ami_id_monitor: 'ami-02eac2c0129f6376b' # Official CentOS Linux 7 x86_64 HVM EBS ENA 1901_01
+    ami_id_monitor: 'ami-01a7efd0776f9a4fe' # scylla_monitoring_2022-09-19T14-12-11Z
```

Comment:
Which version is it of monitor?
I would like it to be clear so every time we have new monitor version (e.g. 4.1).

@amnonh will send a PR that change the branch name and the images IDs.
(We would be nicer if we could read these IDs from a centralized location like we have in scylla-version).

Reply:
I think it's 4.2,

but we take the image for the cached dependencies, not for the monitoring version inside.

we still need to preserve the option to install the monitoring from clean image, cause we manger tests.

Reply:
Installing monitoring will be based on cleanOS.
Wherever we do that (e.g new atifact test for monitor or Azure or manager test).

The rest should use the image as is without installing anything.

This is not high priority at the moment, and you don't need to work on it right now.
(This entire PR).

Reply:
Start a thread with @amnonh so next release would have better names to indicate their branch name